aspnetcore/security/authentication/community.md

```diff
@@ -21,8 +21,9 @@ The list below is sorted alphabetically.
 | Name | Description |
 | ---- | ----------- |
 | [AspNet.Security.OpenIdConnect.Server (ASOS)](https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server) | ASOS is a low-level, protocol-first OpenID Connect server framework for ASP.NET Core and OWIN/Katana. |
+| [Cierge](https://github.com/pwdless/Cierge) | Cierge is an OpenID Connect server that handles user signup, login, profiles, management, and social logins. |
+| [Gluu Server](https://gluu.org/) | Enterprise ready, open source software for identity, access management (IAM), and single sign-on (SSO). For more information, see the [Gluu Product Documentation](https://gluu.org/docs/). |
```

Comment:
Correct me if I'm wrong, but oxAuth is not an ASP.NET Core OAuth2/OIDC server you can install in an ASP.NET Core app. Yet, this list only mentions server projects compatible with ASP.NET Core (at least, that was the original goal).

Reply:
@sbrian26 Can you address @PinpointTownes's remark? ... I was under the impression it would fall in under the ASP.NET Core xplat scenarios ...

> Gluu publishes Linux packages for Ubuntu, CentOS, RHEL and Debian operating systems.